engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/TestInfoProvider.scala

```diff
@@ -59,13 +60,13 @@ class ModelDataTestInfoProvider(modelData: ModelData) extends TestInfoProvider {
       sourceFactory <- extractSourceFactory(source)
       definition = ObjectWithMethodDef.withEmptyConfig(sourceFactory, ProcessObjectDefinitionExtractor.source)
       sourceParams <- prepareSourceParams(definition, source)
-      sourceObj = factory.create[process.Source[Any]](definition, sourceParams)
+      sourceObj = factory.create[process.Source[Any]](definition, sourceParams, Some(Interpreter.InputParamName))
```

Comment:
hopefully in the future we will make it nicer - e.g. source will define its input param? maybe we can add todo for that?

Reply:
Good point - I'll put TODO here